subsys/bluetooth/services/hids.c

```diff
@@ -632,7 +640,7 @@ static ssize_t hids_boot_kb_outp_report_write(struct bt_conn *conn,
 		    .size = sizeof(conn_data->hids_boot_kb_outp_rep_ctx),
 		};
 
-		rep->handler(&report, conn);
+		rep->handler(&report, conn, true);
```

Comment:
Please make it consistent and add a callback to the:
```hids_boot_kb_outp_report_read```

Reply:
done